web/index.html

```diff
@@ -43,16 +43,22 @@ <h2 class="f4">Where do I find my multiaddr?</h2>
             <li class="pb2">
                 <strong>Using IPFS Desktop or IPFS WebUI</strong>
                 <ul>
-                    <li>Open the IPFS WebUI "Status" page via the IPFS Desktop menu or by visiting "http://127.0.0.1:5001/webui" (when using the default config settings)
-                    <li>Click the "Advanced" dropdown to see more detailed node settings</li>
+                    <li>Open the IPFS WebUI "Status" page via the IPFS Desktop menu or by visiting "http://127.0.0.1:5001/webui" (when using the default config settings)</li>
+                    <li>If you want to test your peerID rather than a particular address enter <code>/p2p/{YourPeerID}</code></li>
+                    <li>If you want to test a particular address then click the "Advanced" dropdown to see more detailed node settings</li>
                     <li>Choose an entry in your list of "Addresses" (these are the various multiaddrs via which your node is addressable).
                         Note that some of your addresses may be reachable over the public internet and some (e.g. /ip4/127.0.0.1/tcp/4001/p2p/YourID) will not be

```

Comment:
This could potentially be dropped now since people without the expertise to notice the difference between private IP ranges and public ones can just use peerIDs. Thoughts?

Reply:
Yeah I like that, perhaps also `more detailed node settings` -> `more detailed node information`?

Reply:
Updated